h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -392,6 +392,10 @@ public boolean keepRunning() {
   }
 
   public void pollAndUpdateProgress(Stage stage, String name, long workContribution, Job parentJob, Job subJob, JobType subJobType) {
+    pollAndUpdateProgress(stage, name, workContribution, parentJob, subJob, subJobType, false);
+  }
+
+  public void pollAndUpdateProgress(Stage stage, String name, long workContribution, Job parentJob, Job subJob, JobType subJobType, boolean ignoreTimeout) {
     if (null == subJob) {
```

Comment:
Good call on overloading. The original method is still valid for almost every case.